indexing-service/src/main/java/org/apache/druid/indexing/common/task/CompactionTask.java

```diff
@@ -141,6 +141,8 @@
   @Nullable
   private final Granularity segmentGranularity;
   @Nullable
+  private final GranularitySpec granularitySpec;
```

Comment:
You can remove this since it is no longer used

Reply:
What is no longer used?